mullvad-management-interface/src/types/conversions/access_method.rs

```diff
@@ -219,8 +223,8 @@ mod data {
                 CustomAccessMethod::Socks5(Socks5::Local(Socks5Local { peer, port })) => {
```

Comment:
I think `peer` here should also be renamed `remote_peer`. It's otherwise extremely easy to think that's where you connect to talk to the socks5, which is not actually the case. And `port` -> `local_port`, since we use that in other places. As soon as you have two of anything, you need to describe which one is for what. If `Socks5Local` only described how to connect to the socks5 server it would be fine to call the field `port`. But since it also has to contain more data, also containing ports, we should try to be more explicit IMO